WORKSPACE

```diff
@@ -68,3 +68,13 @@ http_archive(
     ],
 )
 
```

Comment:
Please add a comment above this target identical to the one over rules_pkg (that it's needed as a transitive dependency of @io_bazel)

Reply:
Done.